src/xls.rs

```diff
@@ -330,7 +330,7 @@ impl<RS: Read + Seek> Xls<RS> {
                     0x0017 => {
                         // ExternSheet
                         let cxti = read_u16(r.data) as usize;
-                        xtis.extend(r.data[2..].chunks(6).take(cxti).map(|xti| Xti {
+                        xtis.extend(r.data[2..].chunks_exact(6).take(cxti).map(|xti| Xti {
```

Comment:
When using `chunks_exact` and not divisible by 6 it looks like we'll drop the last 1-5 bytes. What are the consequences of that in this code path? Looking at the mapped code here I don't actually understand what we're trying to accomplish.

Reply:
Right now, the code panics when the next line (334) tries to read an malformed Xti. Since the code currently uses `chunks`, which will return an undersized slice, we would rather ignore the extra bytes than panic.